docs/raph-rover/documentation/getting-started.mdx

```diff
@@ -261,78 +262,13 @@ To access it, open your web browser and navigate to
 [10.10.0.2](http://10.10.0.2).
 
 From here, you can control the rover, access the camera feed, and configure
-various settings. You can control the rover using the on-screen joystick.
+various settings. You can also control the rover using the on-screen joystick.
```

Comment:
This sounds a little strange - the `also` is suggesting that there was one way of controlling mentioned earlier. I would change it to something like this
`From here, you can control the rover using keyboard and on-screen joystick, access the camera feed, and configure various settings.`